app/models/review.rb

```diff
@@ -0,0 +1,6 @@
+class Review < ApplicationRecord
+  belongs_to :movie
+
```

Comment:
If a `user has_many :reviews` I'm assuming we will want to include a `belongs_to :user` here

Reply:
We are planning to add user functionality next week so I removed has_many :reviews from User model to keep consistency.